data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -222,12 +225,38 @@ def _create_tile_data(
             axis=0,
         )
 
+        logger.info("Adding fields for island areas and Puerto Rico")
+        score_tiles[field_names.REGION_FIELD] = np.where(
+            score_tiles[field_names.STATE_FIELD].isin(
```

Comment:
I get nervous about matching on strings -- could we use the FIPS codes instead? Or wdyt? I suppose the downside with matching on FIPS codes is that they're less human readable to find errors, but at the same time, they're more constant and predictable than the string names (which could change capitalization, punctuation, etc).

Reply:
this sounds good, FIPS code works